streaming/index.js

```diff
@@ -1096,6 +1100,17 @@ const startServer = async () => {
         reject(new AuthenticationError('Not authorized to stream this list'));
       });
 
+      break;
+    case 'profile':
+      if (!params.account_id) {
+        reject(new RequestError('Missing account id parameter'));
+        return;
+      }
```

Comment:
What about if the requested `params.account_id` Account has blocked the requestee? Yes, filtering will happen at the per-message level, but we should probably do an authorization check here too.